uikit/src/main/java/com/midtrans/sdk/uikit/views/banktransfer/status/BankTransferStatusPresenter.java

```diff
@@ -37,8 +40,15 @@ public String getVaNumber() {
                     vaNumber = transactionResponse.getPermataVANumber();
                     break;
                 case PaymentType.ALL_VA:
-                    //VA number is based on other VA processor
-                    vaNumber = TextUtils.isEmpty(transactionResponse.getBniVaNumber()) ? transactionResponse.getPermataVANumber() : transactionResponse.getBniVaNumber();
+                    String bank = transactionResponse.getAccountNumbers().get(0).getBank();
+
+                    if (bank.equals(BRI)) {
```

Comment:
please extract logic into private method. please use switch-case and default

Reply:
resolved in the latest commit, please check.